usage.py

```diff
@@ -19,7 +19,8 @@
             handle=".handle",
             defaultPosition={'x': 0, 'y': 100},
             position=None,
-            grid=[10, 10],
+            grid=[12, 12],
+            disabled=True,
```

Comment:
Probably better to keep it enabled by default. Otherwise people trying out usage.py without reading the source code will think it's broken.

Reply:
Fixed